core-web/libs/data-access/src/lib/dot-page-workflows-actions/dot-page-workflows-actions.service.ts

```diff
@@ -0,0 +1,46 @@
+import { Observable } from 'rxjs';
+
+import { Injectable } from '@angular/core';
+
+import { pluck } from 'rxjs/operators';
+
+import { DotRenderMode } from '@dotcms/data-access';
+import { CoreWebService } from '@dotcms/dotcms-js';
+import { DotCMSWorkflowAction, DotCMSContentlet } from '@dotcms/dotcms-models';
+
+export interface DotCMSPageWorkflowState {
+    actions: DotCMSWorkflowAction[];
+    page: DotCMSContentlet;
+}
+
+@Injectable()
+export class DotPageWorkflowsActionsService {
+    constructor(private coreWebService: CoreWebService) {}
```

Comment:
We should use Angular `HttpClient` instead of `coreWebService` services because it's deprecated.

Reply:
@rjvelazco fixed!